### PR TITLE
Add a section detailing some policies around IP infringment.

### PR DESCRIPTION
**Pull request by VanL**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: pep-0541.txt [hunk 1]
```diff
@@ -212,6 +214,39 @@ is considered invalid and will be removed from the Index:
...
 names as unavailable for security reasons.
 
+Intellectual property policy
+----------------------------
+
+It is the policy of Python Software Foundation and the Package Index 
+maintainers to be appropriately responsive to claims of intellectual 
+property infringement by third parties. It is not the policy of
+the Python Software Foundation nor the Package Index maintainers
+pre-screen uploaded packages for any type of intellectual property
```
Comment (pfmoore) [inline pep-0541.txt]: "to pre-screen"

**Comment by tiran** [comment]
@VanL you have to sign the PSF contributor agreement first. Please get in contact with yourself if you have any questions for PSF's designated, trusted lawyer. :)

**Comment by VanL** [comment]
I could have sworn I signed the CLA a long time ago. But I just re-signed.

Lawyers. :/

**Comment by Mariatta** [comment]
@VanL I think you have two accounts in the bugtracker? I saw these two usernames: `vanl` and `VanL`. `vanl` has the CLA signed, but the GitHub username is not linked. 
`VanL` has the GitHub username linked, but CLA not signed.

TIL username in bpo is case sensitive.

**Comment by VanL** [comment]
I could have sworn I signed the CLA a long time ago. But I just re-signed.

Lawyers. :/

**Comment by Mariatta** [comment]
> Is there any way to squash things into one account? Or just delete one?

Hmm I don't know :) Anyone here familiar with how the bugtracker works?

Anyways, CLA check looks good now, thanks!

There's a review comment: https://github.com/python/peps/pull/571#pullrequestreview-94723972

Will you be able to make the requested change?

**Comment by Mariatta** [comment]
Thanks!

**Pull request merged**